libs/common/json_sort.go

```diff
@@ -0,0 +1,31 @@
+package common
```

Comment:
I'd just call the file `json.go`. If you look at other files, you'll see they are named similarly - `int.go`, ...